src/frequenz/sdk/config/_config_managing.py

```diff
@@ -44,7 +72,11 @@ def __init__(
         """Initialize this instance.
 
         Args:
-            config_path: The path to the TOML file with the configuration.
+            config_paths: The paths to the TOML files with the configuration. Order is
+                important, as the configuration will be read and updated in the order
+                of the paths, so the last path will override the configuration set by
+                the previous paths. Dict keys will be merged recursively, but other
+                objects (like lists) will be replaced by the value in the last path.
```

Comment:
Why not appending for lists? I suppose it can get complicated if you want to overwrite a value.. but then the same is true for dict, what if youwant to reset it

Reply:
Because appending to lists makes it impossible to un-append, so there is no way to actually override the previous config file. With `dict` it is more work, but you can override any value by setting it back to its default value.